sys/include/net/ieee802154.h

```diff
@@ -275,6 +275,7 @@ static inline eui64_t *ieee802154_get_iid(eui64_t *eui64, const uint8_t *addr,
             eui64->uint8[0] = addr[i++] ^ 0x02;
             eui64->uint8[1] = addr[i++];
 
+            /* FALLTHROUGH */
```

Comment:
We used "Falls through." in other places (e.g. [here](https://github.com/RIOT-OS/RIOT/blob/master/sys/net/gnrc/sock/gnrc_sock.c#L90)). Could you do so here as well, please.

Reply:
Sure :)

Minor nitpick: `dist/tools/tunslip/tunslip.c` and `dist/tools/tunslip/tunslip6.c` also use FALLTHROUGH comments.

Reply:
Those were imported from third-party, but true :-).